mmjstool/src/index.js

```diff
@@ -72,6 +72,114 @@ const ignored = yargs.
                 () => { /* empty function */ },
                 i18nHandlers.i18nCheckWebapp,
             ).
+            command('clean',
+                'Read the translation file, find all the empty translation strings and remove the translation item',
+                (cleanArgs) => {
+                    cleanArgs.demandOption(['mobile-dir', 'webapp-dir', 'file']).
+                        option('file', {
+                            describe: 'File to remove empty translations from',
+                            default: 'de.json',
```

Comment:
???

Reply:
Do you think it is useless to have it for a single translation file? `de.json` is the alphabetical first.

Reply:
what would be the use case?

Reply:
To try the code on a single file before applying it to all files. But on the other hand we can just revert. So I am 0/5

Reply:
I think it would make sense to check that for the `en.json` file locally, but by default, I don't see it interesting as "default" value. So, I would suggest or remove the default value or change it to `en.json`. By the way, the description is not super clear (probably I would use the full path there, maybe a better wording and an example at the end can help to understand  what is expected).